scenarios/kubernetes_e2e.py

```diff
@@ -610,6 +610,18 @@ def main(args):
             raise ValueError(k8s)
         mode.add_k8s(os.path.dirname(k8s), 'kubernetes', 'release')
 
+    if args.build_federation is not None:
+        if args.build_federation == '':
+            # Empty string means --build-federation was passed without any arguments;
+            # if --build-federation wasn't passed, args.build_federation would be None
+            runner_args.append('--build-federation')
+        else:
+            runner_args.append('--build-federation=%s' % args.build_federation)
```

Comment:
nit: maybe combine the lines? `--build-federation=` is also valid

Reply:
updated.